install.md

```diff
@@ -99,33 +99,56 @@ To install Application Service Adapter:
       -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf "[ SAN ]\nsubjectAltName='DNS:<API-FQDN>'")) \
      -days 365
     ```
-1. Create a `tas-adapter-values.yml` file with the desired installation settings, following the package setting schema.
-
-    You can use the following sample as a template:
-
-    ```yaml
-    ---
-    api_ingress:
-      fqdn: "<API-FQDN>"
-      replicas: <API-REPLICA-COUNT>
-      tls:
-        crt: |
-          <TLS-CRT>
-        key: |
-          <TLS-KEY>
-    package_registry_base_path: "<PACKAGE-REGISTRY-BASE>"
-    kpack_image_tag_prefix: "<KPACK-TAG-PREFIX>"
-    ```
-
-    Where:
-    <ul>
-      <li>`<API-FQDN>` is the FQDN that you want to use for the Application Service Adapter API.</li>
-      <li>`<API-REPLICA-COUNT>` is the desired number of instances for the Application Service Adapter API deployment.</li>
-      <li>`<TLS-CRT>` is the PEM-encoded public certificate for the Application Service Adapter API.</li>
-      <li>`<TLS-KEY>` is the PEM-encoded private key for the Application Service Adapter API.</li>
-      <li>`<PACKAGE-REGISTRY-BASE>` is the container registry "folder"/"project" where application source code (Packages) will be uploaded.</li>
-      <li>`<KPACK-TAG-PREFIX>` is the container registry "folder"/"project" where runnable application images (Droplets) will be uploaded.</li>
-    </ul>
+1. Create a `tas-adapter-values.yml` file with the desired installation settings, following the schema specified for the package.
+
+   The following values are required:
+
+   ```yaml
+   ---
+   api_ingress:
+     fqdn: "<API-FQDN>"
+     tls:
+       crt: |
+         <TLS-CRT>
+       key: |
+         <TLS-KEY>
+   package_registry_base_path: "<PACKAGE-REGISTRY-BASE>"
+   kpack_image_tag_prefix: "<KPACK-TAG-PREFIX>"
+   ```
+
+   Where:
+   - `<API-FQDN>` is the FQDN that you want to use for the TAS adapter API.
+   - `<TLS-CRT>` is the PEM-encoded public certificate for the TAS adapter API.
+   - `<TLS-KEY>` is the PEM-encoded private key for the TAS adapter API.
+   - `<PACKAGE-REGISTRY-BASE>` is the container registry "folder"/"project" where application source code (Packages) will be uploaded
+   - `<KPACK-TAG-PREFIX>` is the container registry "folder"/"project" where runnable application images (Droplets) will be uploaded
```

Comment:
Confirmed, using `-`es for the items still doesn't render correctly in VMware docs:
<img width="854" alt="Screen Shot 2021-12-07 at 12 38 14 PM" src="https://user-images.githubusercontent.com/5836757/145102927-ddb5c041-c7e8-4ec0-9e77-18b598a0bf67.png">